src/client/components/app/VaultDetail/VaultDetailPanels.tsx

```diff
@@ -408,8 +408,8 @@ export const VaultDetailPanels = ({
               <StyledCardHeader header={t('vaultdetails:overview-panel.strategies')} />
 
               <OverviewStrategies>
-                {strategies.map((strategy) => (
-                  <OverviewInfo variant="surface" cardSize="micro">
+                {strategies.map((strategy, index) => (
+                  <OverviewInfo variant="surface" cardSize="micro" key={index}>
```

Comment:
We should avoid using index as key in this case because we do have stable IDs in the form of the strategy address

> When you don’t have stable IDs for rendered items, you may use the item index as a key as a last resort

https://reactjs.org/docs/lists-and-keys.html#keys

```suggestion
                  <OverviewInfo variant="surface" cardSize="micro" key={strategy.address}>
```

Reply:
Addressed with https://github.com/yearn/yearn-finance-v3/pull/565/commits/0f6d97fd2f804a2b5b5450e68e7460eff088ec16